.gitignore

```diff
@@ -21,3 +21,4 @@ coverage.*
 /examples/**/dist
 .history
 .site
+*.tgz
```

Comment:
`npm pack` is an edge case, I don't think that ignore files shouldn't be aware of it